extensions/micrometer/deployment/src/main/java/io/quarkus/micrometer/deployment/MicrometerProcessor.java

```diff
@@ -164,6 +164,11 @@ UnremovableBeanBuildItem registerAdditionalBeans(CombinedIndexBuildItem indexBui
 
         reflectiveClasses.produce(createReflectiveBuildItem(COUNTED_ANNOTATION, index));
         reflectiveClasses.produce(createReflectiveBuildItem(TIMED_ANNOTATION, index));
+        reflectiveClasses.produce(ReflectiveClassBuildItem
```

Comment:
Been a while since I've looked at stuff like this. Does it become a no-op if the classes aren't found?

More a question for myself than a concern with content

Reply:
Those classes are always present. Contained w/in micrometer library.